route-rs-runtime/src/link/primitive/classify_link.rs

```diff
@@ -163,6 +166,8 @@ pub struct ClassifyIngressor<'a, C: Classifier> {
     task_parks: Vec<Arc<AtomicCell<TaskParkState>>>,
 }
 
+impl<'a, C: Classifier> Unpin for ClassifyIngressor<'a, C> {}
```

Comment:
Can we `#[derive(Unpin)]`?

Reply:
Hmm...maybe.

Reply:
Looks like no, that macro doesn't exist...yet?